test/topology_raft_disabled/suite.yaml

```diff
@@ -1,5 +1,5 @@
 type: Topology
-pool_size: 1
+pool_size: 4
```

Comment:
I never understood how someone is supposed to configure this pool_size option... How is 4 better or more correct here than 1 or 14?

Reply:
Indeed. The number of clusters available in the pool should probably depend on the number of cores available on the machine running the tests.

Reply:
@kostja